tests/jballerina-integration-test/src/test/java/org/ballerinalang/test/packaging/PathDependencyTestCase.java

```diff
@@ -603,6 +603,27 @@ public void testRuntimeTimeDependencyForExecutingModuleTests() throws BallerinaT
         buildLogLeecher.waitForText(10000);
     }
 
+    /**
+     * Build TestProject in interop-failure. TestProject has utils module which has an interop error.
+     * When there is an interop error, tests should not be executed.
+     *
+     * @throws BallerinaTestException Error when executing the commands.
+     */
+    @Test(description = "Test whether execution of tests are broken when interop failure exists")
```

Comment:
Can we put this in to a different class since this is not a path dependency test